шахматы/шахматы/Program.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace шахматы
+{
+    internal class Program
+    {
+        static void Main(string[]args)
+        {
+            Console.WriteLine("Введите позицию белого коня");
+            var WhiteKnightPosition = Console.ReadLine();
+            Console.WriteLine("Введите позицию черного коня");
+            var BlackKnightPosition = Console.ReadLine();
+
+            if (!CheckPosition(WhiteKnightPosition) || !CheckPosition(BlackKnightPosition))
+            {
+                Console.WriteLine("Так ставить фигуры нельзя. ");
+                Console.ReadKey();
+
+            }
+            if (KnightUnderStrike(BlackKnightPosition, WhiteKnightPosition))
+            {
+                Console.WriteLine("Кони под ударом друг друга");
+                Console.ReadKey();
+            }
+            Console.WriteLine("Введите ход белого коня");
+            var WhiteKnightMove = Console.ReadLine();
+            if (!CorrectMove(WhiteKnightPosition, WhiteKnightMove))
+            {
+                Console.WriteLine("Конь так не ходит");
+                Console.ReadKey();
+            }
+            else if (WhiteKnightMove == BlackKnightPosition)
+            {
+                Console.WriteLine("Черный конь срублен");
+                Console.ReadKey();
+            }
+            else
+            {
+                if (KnightUnderStrike(BlackKnightPosition, WhiteKnightMove))
+                {
+                    Console.WriteLine("Кони под ударом друг друга");
+                    Console.ReadKey();
+                }
+
+            }
+
```

Comment:
А здесь не хватает сообщения о положительном исходе.
И не стоит в каждой ветке использовать `Console.ReadKey();`. Достаточно написать этот оператор один раз после оператора условия. И код сразу станет проще
```suggestion
            var WhiteKnightMove = Console.ReadLine();

            if (!CorrectMove(WhiteKnightPosition, WhiteKnightMove))
                Console.WriteLine("Конь так не ходит");

            else if (WhiteKnightMove == BlackKnightPosition)
                Console.WriteLine("Черный конь срублен");

            else if (KnightUnderStrike(BlackKnightPosition, WhiteKnightMove))
                Console.WriteLine("Кони под ударом друг друга");

            else
                Console.WriteLine("Безопасный ход");
            
            Console.ReadKey();
```